.github/workflows/jira-status.yml

```
@@ -55,11 +55,18 @@ jobs:
           SAFE_EVENT_NAME: ${{ github.event_name }}
         run: |
           if [ "$SAFE_EVENT_NAME" = "pull_request" ]; then
-            branch_or_pr="$SAFE_HEAD_REF"
+branch_or_pr="$SAFE_HEAD_REF"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 58: could not find expected ':'

(syntax-check)